docs/sharding.md

```diff
@@ -0,0 +1,42 @@
+# Multiple Azure SignalR Service Instances Support
+Currently we support multiple Azure SignalR Service instances **for Persistent mode only**.
+
+## Set up multiple endpoints
+To enable multiple endpoints support, you should: 
+
+1. Set the `ServiceTransportType` to `Persistent` in your configuration.
```

Comment:
Let's state example of the accurate place of configuration at least in head of the document, i.e. `local.settings.json`. Sometimes, customer may not be aware the configuration places.

Reply:
We can assume the reader already read the `README.md` doc and have basic information? Otherwise there will be too much overlap in the documents.

Reply:
Mention it at least once in a document is no bad. I've received a few feedbacks people get lost in these places.